src/data/faq.json

```diff
@@ -2171,66 +2171,299 @@
         "permalink": "Link to this entry",
         "linkToGlossary": "Link to the Glossary",
         "glossaryWords": {
+            "A": [
+                {
+                    "term": "Android",
+                    "anchor": "android",
+                    "description": "Android is one of the two operating systems currently supported by the Corona-Warn-App, along with <a href='#glossary_ios'>iOS</a>, and the most widely used mobile operating system in the world. It is an <a href='#glossary_open_source'>open-source</a> operating system for, for example, mobile (such as smartphones or tables) but also other devices (such as TVs or cars). It is jointly developed by Google and the Open Handset Alliance (OHA)."

```

Comment:
"tables" => "tablets"

Reply:
Suggest changing the sentence to the following to improve readability (and correct the typo):

"It is an <a href='#glossary_open_source'>open-source</a> operating system for mobile devices (such as smartphones or tablets) and other devices (such as TVs or cars)."

Reply:
Will be solved in the next commit, thanks!